test/recipes/20-test_dhparam.t

```diff
@@ -10,7 +10,7 @@
 use strict;
 use warnings;
 
-use OpenSSL::Test qw(:DEFAULT data_file);
+use OpenSSL::Test qw(:DEFAULT data_file cmdstr);
```

Comment:
I assume that was an experiment. Since you're not using cmdstr, this seems unnecessary at this point

Reply:
Yea.. I spent many many hours trying to get a pipe test to work before giving up..